src/Assets/Javascript/get.js

```diff
@@ -0,0 +1,22 @@
+function getAll(iApi) {
+  return new Promise((resolve) => {
+    fetch(iApi)
+      .then((response) => response.json())
+      .then(resolve);
+  });
+}
+
+export async function get(param) {
+  const setVal = await getAll(param);
+  return setVal;
+}
+
+export function getComments(api, array1, array2) {
+  array1.forEach(async (e) => {
+    await fetch(api + e)
+      .then((response) => response.json())
+      .then((json) => {
+        array2.push({ id: e, value: json });
+      });
+  });
+}
```

Comment:
- [ ] Currently when I add a comment data are not recorded and the screen is not updated kindly work on that.